src/shared/import/function/publish.sh

```diff
@@ -0,0 +1,17 @@
+function publish_table {
```

Comment:
Soms moet er ook een en ander gerenamed worden met the constraints voor primary key en geo index die automatisch gecreeerd worden. Je kunt testen of dit goed gaat door de import twee keer achter elkaar te draaien.
De constraints worden niet hernoemd als de tabel wordt hernoemd . Tevens kan het nodig zijn om DRAP TABLE CASCADE te doen

Reply:
Inderdaad, ik krijg wat errors.